tests/functional/object/mcg/test_object_versioning.py

```diff
@@ -139,3 +140,140 @@ def test_versioning_parallel_ops(
             query_out
         ), "[Test failed] There are more than one versions considered to be latest version !!"
         logger.info("Test succeeded!!")
+
+
+@bugzilla("2240714")
+class TestGetObjectByVersionID:
+
+    object_name = "file_1"
+    object_data = "version 1"
+
```

Comment:
Some clarification that these are consts, and why they are not just function-scoped vars:

```suggestion

    # Common consts between the 
    # pre-upgrade and post-upgrade tests
    OBJ_NAME = "file_1"
    V1_OBJ_DATA = "version 1"

```